main/src/com/google/refine/grel/ast/OperatorCallExpr.java

```diff
@@ -78,6 +78,8 @@ public Object evaluate(Properties bindings) {
                     } else if ("/".equals(_op)) {
                         if (n2 == 0 && n1 == 0) {
                             return Double.NaN;
+                        } else if (n2 == 0) {
```

Comment:
This could be simplified so that you're only testing for `n2 == 0` once and then handling all the `n1` cases inside that branch.

Reply:
Okay let me fix that and add the test for the division.